controller/predicates/has_request_parameter.js

```diff
@@ -0,0 +1,3 @@
+const _ = require('lodash');
+
```

Comment:
could we add a single-line code comment which describes the intended purpose, for posterity?